client/actions/checks.js

```diff
@@ -73,9 +79,9 @@ function updateTokenForChecks(repo){
  * Refresh access token with new user logged in (1% case)
  * @param {*} repo - repository data
  */
-export function requestRefreshToken(repo) {
+export function requestRefreshToken(repo, reloadReposFn) {
```

Comment:
Might be interesting to ask someone like @frometor or @ChristianLohmann regarding Redux:
- In the Component I map the action to props and then provide it to my action here. 
-> I don't see really another easy way of triggered a reload of all repo's and the data as the function `reloadIfNeeded` is not bound to any state parameter just function arguments...